src/sentry/models/externalactor.py

```diff
@@ -38,12 +38,10 @@ class Meta:
         unique_together = (("organization", "provider", "external_name", "actor"),)
 
     def delete(self, **kwargs):
-        from sentry.models import NotificationSetting
+        install = self.integration.get_installation(self.organization_id)
 
-        # There is no foreign key relationship so we have to manually cascade.
-        NotificationSetting.objects._filter(
-            target_ids=[self.actor_id], provider=ExternalProviders(self.provider)
-        ).delete()
+        install.notify_remove_external_team(external_team=self, team=self.actor.resolve())
+        install.remove_notification_settings(actor_id=self.actor_id, provider=self.provider)
```

Comment:
@mgaeta  So I was trying to do the noop for these functions but since I've built them into a separate mixin (`src/sentry/integrations/notifications.py`) and import it for the Slack integration (L69 in `src/sentry/integrations/slack/integration.py` on this PR), the noops are overriding the actual functions in the mixin file. Is there a way around this?